rewards/classes/Boost.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass
+from typing import Dict
+from decimal import Decimal
+
+
+@dataclass
+class BoostBalances:
+    native: Dict[str, Decimal]
+    non_native: Dict[str, Decimal]
+    bvecvx: Dict[str, Decimal]
```

Comment:
Are `bvecvx` and `nfts` balance optional? If yes, let's make them optional here, and not pass empty dicts ther